binarySearch.java

```diff
@@ -0,0 +1,43 @@
+//This code is used to find the index of a specified key in a sorted array of integers.
+
+import java.util.Scanner;
+
+public class binarySearch {
```

Comment:
Class name should start with a capital letter in java. 
Please change the class name and the file name to `BinarySearch`.